workshops/geometric_pattern/README.md

```diff
@@ -510,15 +539,15 @@ function draw() {
 
   var isShifted = false;
 
-...
-
+  // ...the rest of the draw function
 }
 ```
 
 And then increment the values at the bottom of the `y` while-loop in `draw()`, by adding these lines.
 
 ```js
-...
+    y = y - circleRadius;
```

Comment:
In some places you're using the `n -= n2` shorthand, but here it's `n = n - n2`... I think it'd be good to keep consistent with that.